.golangci.yml

```diff
@@ -11,7 +11,7 @@ linters-settings:
   golint:
     min-confidence: 0
   gocyclo:
-    min-complexity: 10
+    min-complexity: 20
```

Comment:
Should complexity be increased or should we add a TODO to solve this in the backlog.